automation/src/test/resources/test-grid-is-resources/SolutionPattern22/Tests/jmeter/empty.jmx

```diff
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<jmeterTestPlan version="1.2" properties="3.2" jmeter="3.3 r1808647">
```

Comment:
I think we should still have jmeter based unit tests within testgrid after externalizing the jmeter as well.
Test phase is a core part of testgrid. We need some kind of tool to verify that phase. Jmeter is the ideal candidate for that. 

Our previous impl downloaded jmeter dependency via maven. In the new impl, we are invoking jmeter via script. So, for that, we have to download extract it manually. This can be done via the TestNG @BeforeSuite step.

Shall we do it here or create a separate ticket to track this?

Reply:
+1. New issue [1] created for it.

[1]. https://github.com/wso2/testgrid/issues/733